Yakkity-Yulelogs/reactive-nutshell
# Rc view own events

# Description

User's events now display, with the most recent event first and with CSS per requirements

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all
git checkout rc-view-own-events

confirm events are displaying for user



# Checklist:
